src/evaluation/evaluator.h

```diff
@@ -57,9 +87,8 @@ class Evaluator {
 
             fmt::print("bestmove {}", move);
             if (m_ponderingEnabled) {
-                const auto ponderMove = m_searcher.getPonderMove();
-                if (ponderMove.has_value()) {
-                    fmt::print(" ponder {}", ponderMove.value());
+                if (m_ponderMove.has_value()) {
```

Comment:
rebase conflict

Reply:
I don't think this is just a rebase conflict. `scanForBestMove` does thread voting and returns only returns the best move. From the 'winning' result the ponder move is also needed by `getBestMove`.

We could leave this as is, change the signatures, or secret third thing. What do you think?

Reply:
Agree, I think the current solution is fine. Sorry, was a little too fast there :)